packages/react-aria-components/docs/Autocomplete.mdx

```diff
@@ -116,6 +124,452 @@ import {UNSTABLE_Autocomplete as Autocomplete, SearchField, Menu} from 'react-ar
+Alternatively, a controlled value can be provided using the `inputValue` prop. Note that the input value of the Autocomplete does not affect
+the ComboBox's selected option.
+
+```tsx example
+function Example() {
+  let options = [
+    {id: 1, name: 'Adobe Photoshop'},
+    {id: 2, name: 'Adobe XD'},
+    {id: 3, name: 'Adobe InDesign'},
+    {id: 4, name: 'Adobe AfterEffects'},
+    {id: 5, name: 'Adobe Illustrator'},
+    {id: 6, name: 'Adobe Lightroom'},
+    {id: 7, name: 'Adobe Premiere Pro'},
+    {id: 8, name: 'Adobe Fresco'},
+    {id: 9, name: 'Adobe Dreamweaver'}
+  ];
+  let [value, setValue] = React.useState('Adobe XD');
+
+  return (
+    <div style={{display: 'flex', gap: 16, flexWrap: 'wrap'}}>
+      <MyAutocomplete
+        label="Adobe products (Uncontrolled)"
+        items={options}
+        /*- begin highlight -*/
+        defaultInputValue="Adobe XD">
+        {/*- end highlight -*/}
+        {item => <MyItem>{item.name}</MyItem>}
+      </MyAutocomplete>
+
+      <MyAutocomplete
+        label="Adobe products (Controlled)"
+        items={options}
+        /*- begin highlight -*/
+        inputValue={value}
+        onInputChange={setValue}>
+        {/*- end highlight -*/}
+        {item => <MyItem>{item.name}</MyItem>}
+      </MyAutocomplete>
+    </div>
+  );
+}
+```
+
 ## Props
 
 <PropTable component={docs.exports.UNSTABLE_Autocomplete} links={docs.links} />
+
+## Styling
+
+Since Autocomplete doesn't render any DOM elements itself, it doesn't offer any styling options. See the styling sections for [TextField](TextField.html#styling), [SearchField](SearchField.html#styling), [Menu](Menu.html#styling), and [ListBox](ListBox.html#styling) for more information on how to style components within the Autocomplete.
+
+## Advanced customization
+
+### Composition
+
+If you need to customize one of the components within an `Autocomplete`, such as `TextField`, `SearchField`, `Menu` or `ListBox`, you can create a wrapper
+component. This lets you customize the props passed to the component.
+
+```tsx
+function MyListBox(props) {
+  return <ListBox {...props} className="my-listbox" />
+}
+```
+
+### Contexts
+
+All React Aria Components export a corresponding context that can be used to send props to them from a parent element. This enables you to build your own compositional APIs similar to those found in React Aria Components itself. You can send any prop or ref via context that you could pass to the corresponding component. The local props and ref on the component are merged with the ones passed via context, with the local props taking precedence (following the rules documented in [mergeProps](mergeProps.html)).
+
+<ContextTable components={['UNSTABLE_Autocomplete']} docs={docs} />
+
+This example shows a `FieldGroup` component that renders a group of filters with a title. The entire group can be given the same `filter` prop, which is passed to all child selects via the `AutocompleteContext` provider.
```

Comment:
interesting example 😉 I can't really think of anything else though. Should we even show anything or just list the contexts? Not sure how useful it will be and it does take up some space. 🤔

Reply:
Yeah I couldn't think of a great example here. I'm okay with just listing the contexts since it doesn't seem like a big use case.